modules/shared/config/config.go

```diff
@@ -236,6 +245,12 @@ func (cfg *Config) validate() error {
 func DefaultConfig() *Config {
 	cfg := &Config{}
 
+	// TODO: Bubble this config up somewhere closer to main
+	cfg.OTel.Enabled = true
+	cfg.OTel.Debug = true
```

Comment:
I think Debug should be false by default. That way if/when uers enable it in production they only have to set `Enabled` to `true`.